docs/docs/get_started/quickstart.mdx

```diff
@@ -262,25 +284,116 @@ chain.invoke({"text": "colors"})
 
 Note that we are using the `|` syntax to join these components together.
 This `|` syntax is powered by the LangChain Expression Language (LCEL) and relies on the universal `Runnable` interface that all of these objects implement.
-To learn more about this syntax, read the documentation [here](/docs/expression_language).
+To learn more about LCEL, read the documentation [here](/docs/expression_language).
 
-## Seeing this in LangSmith
+## Tracing with LangSmith
 
 Assuming we've set our environment variables as shown in the beginning, all of the model and chain calls we've been making will have been automatically logged to LangSmith.
 Once there, we can use LangSmith to debug and annotate our application traces, then turn them into datasets for evaluating future iterations of the application.
 
 Check out what the trace for the above chain would look like:
 https://smith.langchain.com/public/09370280-4330-4eb4-a7e8-c91817f6aa13/r
 
+For more on LangSmith [head here](/docs/langsmith/).
+
+## Serving with LangServe
+
+Now that we've built an application, we need to serve it. That's where LangServe comes in.
+LangServe helps developers deploy LCEL chains as a REST API.
+The library is integrated with FastAPI and uses pydantic for data validation.
+
+### Server
+
+To create a server for our application we'll make a `serve.py` file with three things:
```

Comment:
i tend to prefer nouns for file names rather than verbs; so `server.py` rather than `serve.py`